swagger.yml

```diff
@@ -3548,31 +3609,51 @@ paths:
             application/json:
               schema:
                 type: object
-                required:
-                  - Features
-                  - Technologies
-                  - Alerts
                 properties:
                   Features:
-                    $ref: '#/components/schemas/Features' 
-                  Alerts: 
-                    $ref: '#/components/schemas/AggregatedCodeSeverityAlerts'
-                  Technologies:
+                    $ref: '#/components/schemas/Features'
+                  technologies:
                     type: object
-                    required:
-                      - Technologies
-                      - Total
```

Comment:
Why did you have removed this specification?

Reply:
I didn't remove it on purpose. when I changed to #/component/schemas/ I forgot to specify Required. Done !